HTTP Server.core/Server.cs

```diff
@@ -47,30 +46,36 @@ private void ConnectToClient()
         private void HandleData()
         {
             byte[] buffer = new byte[1024];
-
-            ReceiveData(buffer);
-            SendData(buffer);
+            int bytesReceived = ReceiveData(buffer);
+            SendData(HandleReply(buffer,bytesReceived));
```

Comment:
You could say that about any number I give for the size of the array.  The problem is that I don't know how much data is going to be put in the buffer, but I need to initialize it and it has to be an array.